src/tokens/utils.ts

```diff
@@ -80,3 +80,39 @@ function usdBalance(token: TokenBalance): BigNumber {
   // We check that usdPrice is not null before calling this.
   return token.usdPrice!.times(token.balance)
 }
+
+export function convertLocalToTokenAmount({
+  localAmount,
+  tokenInfo,
+  exchangeRates,
+}: {
+  localAmount: BigNumber | null
+  tokenInfo: TokenBalance | undefined
+  exchangeRates: { [token in Currency]: string | null }
+}) {
+  const tokenUsdPrice = tokenInfo?.usdPrice
+  const usdExchangeRate = exchangeRates[Currency.Dollar]
+  if (!tokenUsdPrice || !usdExchangeRate || !localAmount) {
+    return null
+  }
+
+  return localAmount.dividedBy(usdExchangeRate).dividedBy(tokenUsdPrice)
```

Comment:
```suggestion
  const localFiatPerDollar = exchangeRates[Currency.Dollar]
  if (!tokenUsdPrice || !usdExchangeRate || !localAmount) {
    return null
  }

  return localAmount.dividedBy(localFiatPerDollar).dividedBy(tokenUsdPrice)
```
nit: I never remember what the units are of our "exchange rates". How about this rename